CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 5.0.4
```

Comment:
Gonna revert this to "Unreleased" since we can probably ship this together with #204.

```suggestion
## Unreleased
```